src/fondant/__init__.py

```diff
@@ -1,6 +1,12 @@
 import logging
 
+import dask
+
 logging.basicConfig(
     format="[%(asctime)s | %(name)s | %(levelname)s] %(message)s",
     level=logging.INFO,
 )
+
+
+dask.config.set({"dataframe.convert-string": False})
+dask.config.set({"distributed.worker.daemon": False})
```

Comment:
what does this do?

Reply:
This config defines if a `distributed.worker` will run as a daemon/background process or not. 
So in our setup the processes will run in the foreground. I wasn't able to start the processes in the background inside of a docker container. I wasn't able to run the workers of the client as daemon processes. 
We just have to make sure that we apply this configuration before we are initialising the client. 
However, I think this change is indeed just needed for the usage of the LocalCluster/client. I will move it to the `Executer` class right before we initialising the client.